src/app/components/footer/footer.component.scss

```diff
@@ -0,0 +1,14 @@
+.footer {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  height: 88px;
+  background-color: rgb(116 118 122 / 20%);
+}
+
+.content-wrapper {
+  border: 1px solid rgb(255 0 0);
+  width: 100%;
+  height: 100%;
+  max-width: 1200px;
+}
```

Comment:
Need to add rule for extra line for scss file

Reply:
done